_pages/rules.html

```diff
@@ -94,6 +94,7 @@ <h2 class="page-heading">E. Punishments</h2>
     <div class="col-md-12">
       <ol>
         <li>Players may be kicked for breaking the rules. If the rule violation is severe, you may be banned instead.</li>
+        <li>Disrespectful and/or toxic behaviour in chat may result in a mute. The mute length can vary depending on previous offences.</li>
```

Comment:
```suggestion
        <li>Players may be muted for breaking chat rules. If you have been muted before, the mute length may be longer.</li>
```
Something like this instead?

Reply:
Sounds good.